src/index.py

```
@@ -1,41 +1,65 @@
+        # try to fetch actual balance from ord-server if configured
+        try:
+            ord_url = getattr(env, 'ORD_SERVER_URL_MAINNET', None)
+            if ord_url:
+                ord_url = ord_url.rstrip('/') + '/mainnet/wallet-balance'
+                resp = await js.fetch(ord_url)
+                if resp.ok:
+                    # convert JS proxy object to native Python dict
+                    body = await resp.json()
+                    try:
+                        body = body.to_py()
+                    except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)